docker/modules/Dockerfile.faux1

```diff
@@ -8,7 +8,7 @@ FROM daqf/aardvark:latest as java_build
 # Run this separately so it can be shared with other builds.
 RUN $AG update && $AG install openjdk-8-jre
 
-RUN $AG update && $AG install openjdk-8-jdk git
+RUN $AG update && $AG install openjdk-8-jdk git vim iproute2
```

Comment:
take a closer look at the structure of the docker file -- you're messing up the shared java build image by putting this here and it won't end up in the image you think it will.